### ISSUE TITLE
Working on the algorithms to determine prices of houses

### ISSUE THREAD

**Issue by sturdyplum** [pycollabproject/realprice#21]
Basically a method that will take in allot of info about a house and will output a suggested price. It should also be dynamic meaning that it would change depending on the market and new incoming data.

**Comment by calveym** [comment]
@sturdyplum We should do this as a backend-team collaboration, due to the huge scale this would inherently have to take. Looking forward to it!

**Comment by sturdyplum** [comment]
Basically so far my idea is to separate variables into two categories, major variables and minor variables. Major variables should be things like sqr feet amount of rooms etc. minor variables would be things like having a pool. Then for minor variables we can look at average price increase, such as houses with pools are an average of 5000$ more exspensive. So we would add 5000 to the price. Now for major variables it would be better to create a graph of some sort, like and xyz graph in which these variables interact with each other. For example having three rooms wont add as much value if the sqr feet is small because the room will probably be small. But a pool will add a constant value, because a pool does not have too much to do with the sqr feet.

**Comment by calveym** [comment]
Good idea. We would need to store the actual algorithm in a separate file, not necessarily as a DB due to smaller size, so that it can be persistent. Something that we can read in python. Then have a method for calling price, a method for redefining price etc in python file. Thoughts?

**Comment by sturdyplum** [comment]
yeah we can have it update daily, probably would be best to make it its own file like you said.

**Comment by calveym** [comment]
I think it could be interesting to have it update every time the other side of the website is called, with the crawler and price request and all that. Killing two birds with one stone!

**Comment by sturdyplum** [comment]
ok that would work too
